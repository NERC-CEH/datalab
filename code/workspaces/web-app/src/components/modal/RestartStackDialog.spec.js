import React from 'react';
import { shallow } from 'enzyme';
import RestartStackDialog from './RestartStackDialog';

describe('RestartStackDialog', () => {
  const shallowRender = props => shallow(
    <RestartStackDialog {...props} />,
  );

  it('renders to match snapshot', () => {
    const render = shallowRender({
      onSubmit: jest.fn().mockName('onSubmit'),
      title: 'Dialog title',
      body: 'Dialog body',
      onCancel: jest.fn().mockName('onCancel'),
    });

    expect(render).toMatchSnapshot();
  });
});
