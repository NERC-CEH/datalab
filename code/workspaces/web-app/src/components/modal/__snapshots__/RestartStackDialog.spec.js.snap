// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RestartStackDialog renders to match snapshot 1`] = `
<WithStyles(ForwardRef(Dialog))
  maxWidth="md"
  open={true}
>
  <WithStyles(ForwardRef(DialogTitle))>
    Dialog title
  </WithStyles(ForwardRef(DialogTitle))>
  <WithStyles(ForwardRef(DialogContent))>
    <WithStyles(ForwardRef(Typography))
      variant="body1"
    >
      Dialog body
    </WithStyles(ForwardRef(Typography))>
  </WithStyles(ForwardRef(DialogContent))>
  <WithStyles(ForwardRef(DialogActions))>
    <PrimaryActionButton
      icon="check"
      onClick={[MockFunction onSubmit]}
    >
      Confirm
    </PrimaryActionButton>
    <SecondaryActionButton
      icon="clear"
      onClick={[MockFunction onCancel]}
    >
      Cancel
    </SecondaryActionButton>
  </WithStyles(ForwardRef(DialogActions))>
</WithStyles(ForwardRef(Dialog))>
`;
